engine/runtime/src/main/java/org/enso/interpreter/runtime/data/EnsoFile.java

```diff
@@ -45,17 +49,58 @@ public EnsoFile(TruffleFile truffleFile) {
   @Builtin.Method(name = "output_stream_builtin")
   @Builtin.WrapException(from = IOException.class)
   @Builtin.ReturningGuestObject
+  @Builtin.Specialize
   @CompilerDirectives.TruffleBoundary
-  public OutputStream outputStream(OpenOption[] opts) throws IOException {
-    return this.truffleFile.newOutputStream(opts);
+  public OutputStream outputStream(Object opts,
+      EnsoContext ctx,
+      @CachedLibrary(limit = "5") InteropLibrary interop) throws IOException {
```

Comment:
Combination of `@TruffleBoundary` and `@CachedLibrary` - the usual error...

Reply:
Oh, my bad. I will fix that in one of my upcoming PRs - https://github.com/enso-org/enso/commit/ea95a16eb61eb865b6cb7d21b3ca3548a4d542a9